sdk/examples/client/get_block.rs

```diff
@@ -22,7 +22,7 @@ async fn main() -> Result<()> {
         .await?;
 
     // Fetch a block ID from the node.
-    let block_id = client.get_issuance().await?.strong_parents[0];
+    let block_id = client.get_issuance().await?.strong_parents.into_iter().next().unwrap();
```

Comment:
Bit annoying but I guess we don't really have a cleaner way

Reply:
yeah not really